tasks/configure_xen.yml

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
Seems fine, but touching the `xl.conf` and #25 definitely belong into the [xen-setup](https://github.com/stuvusIT/xen-setup) role, please commit it there. I don't know how complex the `xl.conf` template would be, but if you plan to do it in the near future, you could just combine both and not need `lineinfile` tasks at all.

Reply:
This was even my first thought ;). But than I realized that xen-setup does a basic hypervisor setup for xen, such things like installing xen, configure grub on a debian based system. xen_vman instead does everything vm management related, such as creating, installing and manage all vms. Creating a proper xl.conf is needed for the tasks done by xen_vman, so it belongs there.

Future more, you can use xen-setup completely unrelated to the tasks done by xen_vman this way. For example you can setup a hypervisor with xen-setup and later use virtsh to manage all vms on this machine, which will conflict with a xl.conf managed by xen-setup.